crds/resource_crd.go

```diff
@@ -121,39 +105,22 @@ func FromResource(r common.Resource) *ResourceObject {
 		Status: ResourceStatus{
 			Owner:          r.Owner,
 			State:          r.State,
-			LastUpdate:     r.LastUpdate,
-			UserData:       r.UserData,
-			ExpirationDate: r.ExpirationDate,
+			LastUpdate:     v1.Time{Time: r.LastUpdate.UTC()},
```

Comment:
why does `LastUpdate` use UTC but `ExpirationDate` (through `timeToMetaTime()`) does not?

Reply:
That was a remainder, removed it